app/controllers/Dates.php

```diff
@@ -0,0 +1,53 @@
+<?php
```

Comment:
It'd be good to get in the habit of namespacing your classes.
This one would be:
`namespace \App\Controllers;`
Then you can refer to things like: `\App\Controllers\Dates` and `\App\Models\Date`

Reply:
Ye i really havent used namespaces at all, ill give it a go!